README.md

```diff
@@ -61,4 +61,28 @@ $ make run
 ```
 
 You should be able to test the forum at [http://localhost:8090/dev/](http://localhost:8090/dev/).
-This is the development view that emulates a LTI consumer.
\ No newline at end of file
+This is the development view that emulates a LTI consumer.
+
+## Backoffice
+
+You can access the backoffice of the forum at [http://localhost:8090/admin/](http://localhost:8090/admin/)
+
+You first need to create a superuser account. 
+
+```bash
+$ bin/superuser
+```
+
+## Troubleshooting
+
+ElasticSearch service is always down with an Exit 137 State
```

Comment:
Is it also related to the issue we have on richie ? If it's the case you can maybe add more information on how to deal with this error for non mac users: https://richie.education/docs/docker-development#elasticsearch-service-is-always-down

Reply:
I thought about it but actually I didn't have the error on ritchie because it's not the same elasticsearch version so I'm not quiet sure it's exactly relevant

Reply:
To get around it I've change the wording, I've added a N.B and as I don't give concrete solutions for mac users, linux ones won't miss one either. In the Richie doc, we can see error in logs so I didn't want to copy this part because I didn't go through it. The command line for linux is probably right to use but then I will need to be fare to explain how to change it for mac users as well.